### DISCUSSION TITLE
Make dark theme available on Marketplace already

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#76868] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

It has been several years now since GitHub shipped a dark theme (And later on additional ones for better a11y), yet the marketplace still has no support for dark theme whatsoever.

Every time I visit it am I blinded by the extreme white background that welcomes me, which actually even hurts my eyes.
Using an extension such as Dark reader isn't a good aproach for me as it would apply to the whole domain, affecting github itself as a whole. And my knowledge of this extension is too small to make a filter that only applies to marketplace.

Either way, there is really no excuse to not have dark theme already on the marketplace after it has been available on GitHub for several years now.

**Comment by marifcelik** [comment]
There is a solution for this. Actually there is a dark theme for the marketplace, but I think there is a bug. If you open the developer console and add these properties to the `html` element, page should switch to the dark theme.
```
data-color-mode="dark" data-dark-theme="dark"
```

I think a browser add-on can created for this.

**Reply by Andre601** [reply]
That's interesting...

Tho, it now makes we wonder why this hasn't been fixed yet if it is a known bug? I really doubt that this is a difficult bug to even fix here...

**Comment by pabloqpacin** [comment]
Yes please, this feature is quite a must-have 🙏

**Comment by Andre601** [comment, marked as answer]
It seems GitHub fixed the issue with marketplace not using dark theme, as it now seems to use it whenever I go to it.

My eyes are thankful for this change. 👍
